documentation/src/main/markdown/currentreleasenotes.md

```diff
@@ -16,4 +16,8 @@
 ### Resolved issues
 * (IDETECT-4056) Resolved an issue where no components were reported by CPAN detector.
   If the cpan command has not been previously configured and run on the system, [solution_name] instructs CPAN to accept default configurations.
-* (IDETECT-3843) Additional information is now provided when [solution_name] fails to update and [solution_name] is internally hosted.
\ No newline at end of file
+* (IDETECT-3843) Additional information is now provided when [solution_name] fails to update and [solution_name] is internally hosted.
+
+### Dependency updates
+* Upgraded Alpine version of Detect's Docker image to 3.18 to allow pulling the latest curl version with no known vulnerabilities.
+* Updated Detect's Ubuntu Docker image to use wget instead of curl to remove curl as a dependency.
```

Comment:
Suggested sentence reorg. ; )

* Removed curl as a dependency by updating [solution_name] Ubuntu Docker image to use wget instead of curl.

Reply:
Just to note, curl is being removed only from Ubuntu's Docker image. Should we say 
```
* Removed curl as a dependency from [solution_name] Ubuntu Docker image by using wget instead of curl.
```

Reply:
Yes. Thank you.